experimental/PropertyDDS/services/property-query-service/package.json

```diff
@@ -100,14 +101,12 @@
 		"nyc": "^15.0.0",
 		"prettier": "~2.6.2",
 		"promisify": "0.0.3",
-		"request": "^2.88.0",
-		"request-promise-native": "1.0.7",
 		"rimraf": "^2.6.2",
 		"sinon": "^7.4.2",
 		"sinon-chai": "3.3.0",
 		"stats-percentile": "3.1.0",
 		"typescript": "~4.5.5",
-		"wait-on": "3.2.0",
+		"wait-on": "7.0.1",
```

Comment:
This is likely a breaking change? Can we guarantee there will be no behavior changes if we import major bump? https://github.com/microsoft/FluidFramework/wiki/Breaking-vs-Non-breaking-Changes